examples/image-builder.yaml

```diff
@@ -49,36 +53,52 @@ deployment_groups:
     source: modules/packer/custom-image
     kind: packer
     settings:
-      disk_size: 20
```

Comment:
nit: I think I prefered when the disk_size was explicitly tied to the packer module. It is easy to miss now that it is being propagated down (not so much for node group since you make an explicit reference).

One option would be to set disk size to 32 here and just let the node group use the default of 50.

Reply:
I share your point about being explicit. I'm going to add:
```
kind: packer
...
  disk_size: $(vars.disk_size)
```
But I will keep the same disk_size for other machines to "save customers many"